### PR TITLE
test: tck tests now check that AND and OR operators are short circuit evaluated

### PR DESCRIPTION
**Pull request by Cali0707**
This is a follow up on #1267 and #1270, to make sure that those changes are included in the tck tests.

## Proposed Changes

- Test that the OR operator is short circuit evaluated
- Test that the AND operator is short circuit evaluated

### CODE REVIEW

**Comment by Cali0707** [comment]
cc @pierDipi @duglin

File: cesql/cesql_tck/binary_logical_operators.yaml [hunk 1]
```diff
@@ -25,6 +28,9 @@ tests:
   - name: True or true
     expression: TRUE OR TRUE
     result: true
+  - name: OR operator is short circuit evaluated
+    expression: "true and (1 != 1 / 0)"
```
Comment (duglin) [inline cesql/cesql_tck/binary_logical_operators.yaml:32]: should this be "OR" instead of "AND" ?

Comment (Cali0707) [reply cesql/cesql_tck/binary_logical_operators.yaml:32]: Whoops yes, copy paste error 😅

**Comment by duglin** [comment]
LGTM

**Comment by duglin** [comment]
Approved on the 5/23 call

**Pull request merged**